<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIPS Debugger</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_MIPS_Debugger">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MIPS Debugger</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clock">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock Pulse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_File_Directory">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label__File_Directory">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File Directory:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_I_Memory_Read">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Instruction Memory Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_I_Memory_Program">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Instruction Memory Program</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Value_from_I_Memory">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Run">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>590</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Read_Reg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Reg_number">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>620</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 10:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 7:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 6:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 8:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 9:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg15">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 15:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg20">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 20:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 12:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 11:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg19">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 19:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg16">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 16:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg18">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 18:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg13">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 13:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg17">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 17:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg14">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg25">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 25:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg30">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 30:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg22">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 22:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg21">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 21:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg29">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 29:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg26">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 26:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg28">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 28:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg23">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 23:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg27">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 27:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg24">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 24:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Reg31">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Register 31:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Read_All_Registers">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Read All Registers</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
